gateway/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+type docker &> /dev/null || { echo "Docker is not instaled"; exit 1; }
+
+if !docker info &> /dev/null; then
+    cat <<EOF
+Docker is not running or your current user is not in docker group.
+You need to manually add your current user to docker group or run this installer using sudo.
+
+EOF
+    while true; do
+	echo -n "Do you want to run the installer using sudo? [Y/n] "
+	read yn
+	case $yn in
+            [Yy]*) SUDO="sudo"; break;;
+            [Nn]*) exit;;
```

Comment:
When using `no`, it should print something so the user knows it won't be run. Empty string should be considered as `no`.

Reply:
Quando a string é vazia entra a opção default, não faz sentido